guides-markdown/op_return.md

```diff
@@ -0,0 +1,47 @@
+# Create an OP_RETURN output transaction
+
+```post-author
+Javed Khan
+```
+
+OP_RETURN is a script opcode which can be used to store an arbitrary 40-byte
+data as a part of the signature script (null data script), allowing one to
+embed small amount of data into the blockchain. For example, it can be used as
```

Comment:
I think it should probably be "embed small amounts of data" or "embed a small amount of data"

Reply:
Thanks, fixed.